test/apps/iodemo/io_demo.cc

```diff
@@ -586,6 +598,7 @@ class DemoClient : public P2pDemoCommon {
             info.push_back(op_info);
         }
 
+        int conn_num = IoDemoRandom::rand(0, opts().num_servers - 1);
```

Comment:
we should use a random connection for each message.
the way you wrote it now, it uses just one server for all messages

Reply:
@yosefe Using random connection for each message cause bad performance. The throughput is about 800MB/s. Current way is about 3000MB/s.

Reply:
@yosefe Above test result is for read op with two servers. I set -w 1024.
After change to -w 32 don't see so big performance drop. Currently only has about 300MB performance drop with two servers.

Reply:
@yosefe Fixed.